### DISCUSSION TITLE
Borg backup feature request

### DISCUSSION THREAD

**Discussion by infectedw** [pterodactyl/panel#4715] [Feature Requests]
Hello, I use Pterodactyl for my hosting company and the main problem are backups since S3 on Pterodactyl is unstable, it would be awesome if we could get a feature like Borg backups.

BorgBackup is a deduplicating backup program. Borg could help us to save space much faster, efficiently, and securely.

Docs about Borg: 

https://www.borgbackup.org/

https://borgbackup.readthedocs.io/en/stable/

Thanks.

**Comment by parkervcp** [comment]
You should probably have that configured separately from pterodactyl anyways.

**Reply by infectedw** [reply]
Borg also supports "Repository", so you can literally host a borg server on Storage Machine and set that Borg on the client machine send everything to the storage machine. Much better and much more stable than S3.
